pkg/utils/nodepool/nodepool.go

```diff
@@ -16,103 +16,19 @@ package nodepool
 
 import (
 	"context"
-	"fmt"
-	"time"
 
-	"github.com/samber/lo"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
-	"github.com/aws/karpenter-core/pkg/apis/v1alpha5"
 	"github.com/aws/karpenter-core/pkg/apis/v1beta1"
-	provisionerutil "github.com/aws/karpenter-core/pkg/utils/provisioner"
 )
 
 type Key struct {
 	Name          string
 	IsProvisioner bool
 }
 
-func New(provisioner *v1alpha5.Provisioner) *v1beta1.NodePool {
```

Comment:
Can't drop this yet since the `karpenter-convert` tool is still relying on it.

Reply:
Ah tragedy